key-terms.md

```diff
@@ -1,211 +1,194 @@
+</div>
 
 ### User Agent
 
 WCAG 2 defines **user agent** as:
 
-><DL><DT>user agent</DT><DD>
-> 
 > any software that retrieves and presents Web content for users
-> 
-</DD></DL>
+
 <div class="example">
+	<p>Web browsers, media players, plug-ins, and other programs—including [assistive technologies](#dfn-assistive-technology) &mdash; that help in retrieving, rendering, and interacting with Web content.</p>
+</div>
 
->Web browsers, media players, plug-ins, and other programs—including [assistive technologies](#dfn-assistive-technology)—that help in retrieving, rendering, and interacting with Web content.</div>
-For non-web ICT, “user agent” needs to be viewed differently. In WCAG 2, the term “user agent” only refers to retrieval and display of web content. For non-web ICT, the term “user agent” refers to retrieval and display of separate content that is _not on the Web_, which WCAG2ICT refers to as a “document”. Within WCAG2ICT, the term “user agent” is used as follows:
+For non-web ICT, “user agent” needs to be understood differently.  In WCAG 2, the term “user agent” only refers to retrieval and display of web content.  For non-web ICT, the term “user agent” refers to retrieval and display of separate content that is _not on the Web_, which WCAG2ICT refers to as a “document”.
 
-<DL><DT>user agent (as used in WCAG2ICT)</DT><DD>
+Within WCAG2ICT, the term **user agent** is used as follows:
 
-any [software](#software) that retrieves and presents **<INS>[documents]</INS>** for users
-</DD></DL>
-<div class="note">
+> any [software](#software) that retrieves and presents **<ins>\[documents]</ins>** for users
 
-Software that only displays the [content](#content-on-and-off-the-web) contained within it is not considered to be a user agent. It is just considered to be software.</div>
 <div class="note">
-
-An example of software that is not a user agent is a calculator application that doesn't retrieve the calculations from outside the software to present it to a user. In this case, the calculator software is not a user agent, it is simply software with a user interface.</div>
+	<p>Software that only displays the [content](#content-on-and-off-the-web) contained within it is not considered to be a user agent.  It is just considered to be software.</p>
+</div>
+<div class="example">
+	<p>An example of software that is not a user agent is a calculator application that doesn’t retrieve the calculations from outside the software to present it to a user.  In this case, the calculator software is not a user agent, it is simply software with a user interface.</p>
+</div>
 <div class="note">
-
-Software that only shows a preview of content such as a thumbnail or other non-fully functioning presentation is not providing user agent functionality.</div>
+	<p>Software that only shows a preview of content such as a thumbnail or other non-fully functioning presentation is not providing user agent functionality.</p>
```

Comment:
Paragraph markdown isn't needed if you add an extra blank line between the beginning div with the class of "note" and the next line. This makes the document revert back to using markdown.

Reply:
Great!  My technology stack won't put P inside DIV unless I use HTML for everything meant to be inside the DIV.